src/Microsoft.VisualStudio.Editors/Common/Utils.vb

```diff
@@ -1566,6 +1566,56 @@ Namespace Microsoft.VisualStudio.Editors.Common
         ''' <remarks></remarks>
         Friend Function IsTargetingDotNetFramework45OrAbove(hierarchy As IVsHierarchy) As Boolean
 
+            Dim frameworkName As FrameworkName = Nothing
+            If TryGetTargetFrameworkMoniker(hierarchy, frameworkName) Then
+                If String.Compare(frameworkName.Identifier, ".NETFramework", StringComparison.OrdinalIgnoreCase) = 0 AndAlso
+                (frameworkName.Version.Major > 4 OrElse (frameworkName.Version.Major = 4 AndAlso frameworkName.Version.Minor >= 5)) Then
+                    ' Targeting .NET framework 4.5 or higher
+                    Return True
+                ElseIf IsTargetingDotNetCore(frameworkName.Identifier) Then
```

Comment:
The main problem is that when the pages were made it was presuming that there wouldn't be anything other than .Net framework for those.

With that in mind this is internal for the utils and and only used for the Prefer32Bit check. The other option here is to change the check in the property page to call both functions.